index.src.html

```diff
@@ -836,9 +839,9 @@ <h4 id="algorithm-create" algorithm>Create a `Credential`</h4>
     7.  Return |p|.
   </ol>
 
-  <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
+  <h4 id="algorithm-require-mediation" algorithm>Prevent Silent Access</h4>
 
-  The <dfn abstract-op>Require user mediation</dfn> algorithm accepts an [=environment settings
+  The <dfn abstract-op>Prevent Silent Access</dfn> algorithm accepts an [=environment settings
   object=] (|settings|), and returns a {{Promise}} which resolves once the `requires user mediation`
```

Comment:
I agree with @foolip. If we're going to change the verbage, we need to change the flag as well along with the "|origin| requires user mediation" definition.